File: src/main/java/net/maple/handlers/DoNothingHandler.java
package net.maple.handlers;

import client.Client;
import util.packet.PacketReader;

public class DoNothingHandler extends PacketHandler {

    @Override
    public void handlePacket(PacketReader reader, Client c) {
        // ...
    }

    @Override
    public boolean validateState(Client c) {
        return true;
    }
}

File: src/main/java/net/maple/handlers/DoNothingHandler.kt
package net.maple.handlers

import client.Client
import util.packet.PacketReader

class DoNothingHandler : PacketHandler {

    override fun handlePacket(reader: PacketReader, c: Client) {
        // ...
    }

    override fun validateState(c: Client): Boolean {
        return true
    }
}